src/client/Microsoft.Identity.Client/AppConfig/ManagedIdentityApplicationOptions.cs

```diff
@@ -10,19 +12,8 @@ namespace Microsoft.Identity.Client
     public class ManagedIdentityApplicationOptions : BaseApplicationOptions
```

Comment:
Top of mind thought - it's kind of confusing that there're `ManagedIdentityApplicationOptions` and `ManagedIdentityConfiguration`; and both are public. Our other AppOptions classes are public while AppConfig classes are internal.
Feels like the Options class should include the functionality of the config class.

Reply:
Yeah, I agree. There should only be 1 class that is responsible for configuring the application.

Reply:
As discussed in the standup, removing the ManagedIdentityApplicationOptions for now and will add as needed later.